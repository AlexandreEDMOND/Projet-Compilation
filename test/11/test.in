int main()
{
  int a = 0;

  while(a < 10){
    printf("Affichage\n");
    a++;
  }
  print(a);
  
  return 0;
}