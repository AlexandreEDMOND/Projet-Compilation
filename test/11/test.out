Affichage
Affichage
Affichage
Affichage
Affichage
Affichage
Affichage
Affichage
Affichage
Affichage
10
